libkbfs/tlf_edit_history.go

```diff
@@ -528,6 +556,36 @@ func (teh *TlfEditHistory) updateHistory(ctx context.Context,
 	}
 
 	currEdits.addNewEdits(newEdits)
+	// Send the notifications.
+	for writer, edits := range newEdits {
+		for _, edit := range edits {
+			var n *keybase1.FSNotification
+			switch edit.Type {
+			case FileCreated:
+				cop, ok := edit.op.(*createOp)
+				if !ok {
+					return fmt.Errorf("No create op for create notification, "+
```

Comment:
Though I guess a warning line and a `continue` here and below would make sense, so we don't lose valid notifications.  Will do.